eclass/kernel-install.eclass

```diff
@@ -598,6 +599,12 @@ kernel-install_pkg_preinst() {
 		rm "${ED}/lib/modules/${release}"/{build,source} || die
 		dosym "../../../src/linux-${dir_ver}" "/usr/lib/modules/${release}/build"
 		dosym "../../../src/linux-${dir_ver}" "/usr/lib/modules/${release}/source"
+		for file in vmlinux vmlinuz; do
+			if [[ -e "${ED}/lib/modules/${release}/${file}" ]]; then
```

Comment:
I would change this to [[ -L ... ]] to check for a symlink explicitly. That avoids the check failing if the symlink is broken somehow.

Reply:
That makes sense, Fixed!